src/components/ButtonsBox.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] - [ ] Please kindly do not disable linters errors, fix linters helps programmers to write good and clean codebase. Kindly fix your errors 🙃 😉